File: src/types/review.ts
import type { RiskLevel } from './workspace';

export interface ReviewItem {
  id: string;
  workspaceId?: string;
  workspaceName: string;
  repo: string;
  branch: string;
  risk: RiskLevel;
  filesChanged: number;
  additions: number;
  deletions: number;
  aiSummary: string;
  author: string;
  createdAt: string;
}

File: frontend/src-tauri/src/models/review.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Serialize, Deserialize)]
#[serde(rename_all = "camelCase")]
pub struct ReviewItem {
    pub id: String,
    pub workspace_id: String,
    pub workspace_name: String,
    pub repo: String,
    pub branch: String,
    pub risk: String,
    pub files_changed: u32,
    pub additions: u32,
    pub deletions: u32,
    pub ai_summary: String,
    pub author: String,
    pub created_at: String,
}
